CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Breaking. Remove FeatureType enum, replace with np.dtype. FeatureType.BINARY -> np.uint8, FeatureType.FLOAT -> np.float32, FeatureType.INT64 -> np.int64.
 
+- Breaking. Rename get_feature_type -> get_python_type.
```

Comment:
Wrong section, we already released 0.1.56. It should be in the unreleased section.

Reply:
No this is an old change that I forgot to document, it is int he 0.1.56.